Source/ACE.Entity/AcePortalObject.cs

```diff
@@ -4,22 +4,19 @@
 namespace ACE.Entity
 {
     [DbTable("vw_ace_portal_object")]
-    [DbGetList("vw_ace_portal_object", 14, "baseAceObjectId")]
+    [DbGetList("vw_ace_portal_object", 14, "AceObjectId")]
     public class AcePortalObject : AceObject
     {
```

Comment:
Yea - I was just trying to get what we had working again - the only thing special about portals aside from other objects that can have position is they have both a position and a destination.   That can be done in generic handling code for sure.